### PR TITLE
React rewrite

### PR DESCRIPTION
**Pull request by Niputi**

### CODE REVIEW

File: .archive/views/donate.pug [hunk 1]
```diff
@@ -0,0 +1,65 @@
+extends templates/base
```
Comment (velddev) [inline .archive/views/donate.pug]: Make sure to just remove the .archive folder. There's no reason for it to be in the repo anymore if the site has been fully ported.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,8 @@
-# Miki.Web.Frontend
-The official repository for Miki.ai's frontend code. Collaborate and post issues here related to the current website version.
+# Miki.ai
 
-## Running with just parcel
-Standard dev mode with built-in parcel webserver
+https://miki.ai
```
Comment (velddev) [inline README.md]: Maybe a bit more of an explanation for why this repository exists would be nice

File: gatsby-config.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    {
+      resolve: `gatsby-plugin-manifest`,
+      options: {
+        name: "Miki: the Discord bot!",
+        short_name: "Miki",
+        theme_color: "#f23084",
+        background_color: "#2196f3",
+        display: "standalone",
+        orientation: "portrait",
+        icon: "static/miki_avatar.png"
```
Comment (velddev) [inline gatsby-config.js]: Consider moving static content to the CDN when possible.

File: react-conditions.d.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
+declare module "react-condition" {
```
Comment (velddev) [inline react-conditions.d.ts]: Consider seeing if we can push a PR to the [@types repository](https://github.com/DefinitelyTyped/DefinitelyTyped) :-) Would be cool to see that Miki boys are contributing where we can

File: src/pages/donate.tsx [hunk 5]
```diff
@@ -0,0 +1,158 @@
+import * as React from "react"
+import { RouteComponentProps } from "@reach/router"
+import NavbarComponent from "../components/navbar"
+import { FooterComponent } from "../components/footer"
+// import RewardComponent from "../components/cards/reward"
```
Comment (velddev) [inline src/pages/donate.tsx]: Remove comments that are not being used

File: src/pages/donate.tsx [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+import donate1 from "../img/donate/donate_1.png"
```
Comment (velddev) [inline src/pages/donate.tsx]: Consider moving images to the CDN

File: src/styles/style.scss [hunk 7]
```diff
@@ -31,6 +31,16 @@ a.about-link {
 	line-height: 3.25rem;
 }
 
+.is-green {
```
Comment (velddev) [inline src/styles/style.scss]: What's this used for? we have `$is-danger` `$is-white` and `$is-success` already

File: .archive/views/donate.pug (hunk 1 above)
Comment (Niputi) [reply .archive/views/donate.pug]: that is already gone

File: react-conditions.d.ts (hunk 4 above)
Comment (Niputi) [reply react-conditions.d.ts]: I already have a pr waiting to be merged https://github.com/andrewfluck/react-condition/pull/3

File: src/styles/style.scss (hunk 7 above)
Comment (Niputi) [reply src/styles/style.scss]: seem like nohwere

Comment (Niputi) [reply src/styles/style.scss]: gone now

File: src/pages/donate.tsx (hunk 5 above)
Comment (Niputi) [reply src/pages/donate.tsx]: gone now

**Pull request merged**